docs/guide/creating-a-service.md

```diff
@@ -1,14 +1,45 @@
 # Creating a service
 
-`cd` into a directory of your choice and run `serverless create --name my-service --provider aws`.
-Serverless will create a skeleton for your new Serverless service inside the `my-service` directory.
+Let's create our first Serverless service!
+
+## Creating our service
+
+`cd` into a directory of your choice and run `serverless create --name my-service --provider aws` to create a service
+skeleton with the service name `my-service` on your local machine.
+
+Services are provider specific. The `--provider aws` tells Serverless to create an Amazon Web Services dependent service.
 
 Type `cd my-service` to navigate into the previously created directory.
 
-## Open the service inside an editor
+## Open the service inside your editor
+
+Let's take a closer look at the skeleton Serverless has created for us. Open up the `first-service` directory with your
+favorite editor.
+
+You'll see the following files:
+- `serverless.yaml`
+- `serverless.env.yaml`
+- `handler.js`
+
+### [`serverless.yaml`](../understanding-serverless/serverless-yaml.md)
+
+This is our core service file. You can see the name of our service, the provider and the first function inside the
+`functions` definition which points to the `handler.js` file.
```

Comment:
There should also be a link here that goes the the specific definition (same as in the header) but specifically says that it goes to it, e.g. "If you want to read more about the details of your serverless.yml check out our [in-depth guide](../understanding-serverless/serverless-yaml.md)."